index.html

```diff
@@ -37,7 +37,144 @@ <h3 id="foot">LET’S CONNECT</h3>
+      </section>
+    
+      <section class="card">
+        <img src="images/second.png" alt="" aria-hidden="true">
+        <h2> Multi-Post <br> Stories </h2>
+        <div class="exp">
+          <h3>CANOPY</h3>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">Back End Dev</p>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">2015</p>
+        </div>
+        <p class="par">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="programming">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button type='button'>See Project</button>
+      </section>
+    
+      <section class="card">
+        <img src="images/thired.png" alt="">
+        <h2>Tonic</h2>
+        <div class="exp">
+          <h3>CANOPY</h3>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">Back End Dev</p>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">2015</p>
+        </div>
+        <p class="par">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="programming">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <div>
+          <button type='button'>See Project</button>
+        </div>
+      </section>
+    
+      <section class="card">
+        <img src="images/forth.png" alt="" aria-hidden="true">
+        <h2>Multi-Post <br> Stories</h2>
+        <div class="exp">
+          <h3>CANOPY</h3>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">Back End Dev</p>
+          <img src="images/gc3.jpg" alt="">
+          <p class="bullet">2015</p>
+        </div>
+        <p class="par">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="programming">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <div>
+          <button type='button'>See Project</button>
+        </div>
+      </section>
+    </div>
+    
+    <!--About me-->
+    <div>
+      <section class="about">
+        <h2>About <br> Myself</h2>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have 
+            a project you need coded, don’t hestiate to contact me.</p>
+        <h3 id="connect">LET'S CONNECT </h3>
+        <div class="soc">
+          <i class="fa-brands fa-twitter"></i>
+        <i class="fa-brands fa-linkedin-in"></i>
+        <i class="fa-duotone fa-m"></i>
+        <i class="fa-brands fa-github"></i>
+        <i class="fa-brands fa-angellist"></i>
+        </div>
+        <img src="images/social-media-icons.png" alt="" aria-hidden="true">
+        <button type='button'>Get my resume</button>
```

Comment:
## As asked by the previous Code Reviewer, Please work on the button and make it look similar to the Figma Design. 👍🏻 
### Your Design
<img width="199" alt="Screen Shot 2022-04-29 at 5 47 40 PM" src="https://user-images.githubusercontent.com/83048208/165947151-fa37b858-61af-4c02-be04-1af245856d78.png">

### Required Design
<img width="193" alt="Screen Shot 2022-04-29 at 5 48 02 PM" src="https://user-images.githubusercontent.com/83048208/165947184-9ca0ad16-f751-4948-a721-f2b45e4c00bc.png">